### DISCUSSION TITLE
Sugerencia para puntuación

### DISCUSSION THREAD

**Discussion by matiuri** [danirod/rectball#125] [Ideas]
Se me ocurren dos ideas para la cantidad de puntos de una combinación:
1. Que se añadan puntos extras por cada bola dentro de la combinación que tenga el mismo color que las esquinas.
2. Que se cuenten también las combinaciones **COMPLETAS** que se encuentren dentro del área de combinación, incluyendo las esquinas. Algo así como combinaciones anidadas.

**Comment by danirod** [comment]
Lo voy a retrasar para la versión de la semana que viene porque se me ha ocurrido una forma interesante de aplicar tu segunda recomendación y quiero hacerlo con calma.
